snuba/admin/audit_log/base.py

```diff
@@ -10,7 +10,9 @@
 
 class AuditLog:
     def __init__(self) -> None:
-        self.logger = structlog.get_logger()
+        self.logger = structlog.get_logger().bind(
+            module=self.__class__.__module__, context_class=self.__class__.__qualname__
```

Comment:
what does the `context_class` give us? is the module not enough?

Reply:
module is enough. anything to trigger the binding would work